platform/linux-generic/odp_stash.c

```diff
@@ -267,22 +306,15 @@ int odp_stash_destroy(odp_stash_t st)
 {
 	stash_t *stash;
 	int index;
-	odp_shm_t shm;
 
 	if (st == ODP_STASH_INVALID)
 		return -1;
 
 	stash = (stash_t *)(uintptr_t)st;
 	index = stash->index;
-	shm   = stash->shm;
 
 	free_index(index);
```

Comment:
Now that only `index` is needed, perhaps directly `free_index(((stash_t *)(uintptr_t)st)->index)`.

Reply:
Fixed in V2.